pkg/cli/mirror/release.go

```diff
@@ -342,3 +365,95 @@ func (d downloads) Merge(in downloads) {
 		d[k] = v
 	}
 }
+
+//go:embed release-configmap.yaml
+var b []byte
+
+func (o *ReleaseOptions) generateReleaseSignatures(releaseDownloads downloads) error {
+
+	httpClientConstructor := sigstore.NewCachedHTTPClientConstructor(o.HTTPClient, nil)
+
+	manifests, err := manifest.ParseManifests(bytes.NewReader(b))
+
+	if err != nil {
+		return err
+	}
+
+	// Attempt to load a verifier as defined by the release being mirrored
+	imageVerifier, err := verify.NewFromManifests(manifests, httpClientConstructor.HTTPClient)
+
+	if err != nil {
+		return err
+	}
+
+	for image, _ := range releaseDownloads {
+		digest := strings.Split(image, "@")[1]
+
+		ctx, cancelFn := context.WithCancel(context.Background())
+		defer cancelFn()
+		if err := imageVerifier.Verify(ctx, digest); err != nil {
+			// This may be a OKD release image hence no valid signature
+			logrus.Debugf("An image was retrieved that failed verification: %v", err)
```

Comment:
I would suggest doing a continue instead of a break in case valid OCP release exists in `releaseDownloads` after one that failed verification.
```suggestion
			logrus.Warnf("An image was retrieved that failed verification: %v", err)
			continue
```

Reply:
Ack.